src/components/Header/WhatNew.tsx

```
@@ -132,11 +143,12 @@ const WhatNew: React.FC = () => {
             date="2023-11-20"
             li={[
               <>
-                add article:
-                <Link href="https://laststance.io/articles/Trying-less-energy-coding">
-                  Trying less energy coding
-                </Link>
-                '
+                add article:{' '}
+                <DialogClose asChild>
+                  <Link href="/articles/Trying-less-energy-coding">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)